src/baseSwagger.ts

```diff
@@ -229,16 +216,23 @@ function generateEndpointSwaggerSchema(endpoint: IEndpoint, sharedComponents: Co
 					description = description.replace('@deprecated', '').trim()
 				}
 
-				let permissionDescription = ''
-				if (config.permissions) {
-					if (config.permissionsDescriptionFormatter && typeof config.permissionsDescriptionFormatter === 'function') {
-						permissionDescription = config.permissionsDescriptionFormatter(permissions)
+				let middlewaresDescription = ''
+				config.middlewares?.forEach((configMiddleware) => {
+					const endpointMiddleware = middlewares.find((middleware) => middleware.name === configMiddleware.middlewareName)
+					const middleware: IMiddleware = {
+						closure: configMiddleware.closure,
```

Comment:
@kot-1999 what do you need this closure for?

Reply:
@daniel-zavacky It depends on user. If user will need that info during formatting, it will be awailable at least. Real situation when it will be realy needed is when two or more middlewares will've got the same middlewareName. In this case closure will help in better identification.